_src/field-attrs.md

```diff
@@ -77,7 +77,7 @@
 - ##### `#[serde(borrow)]`
 
   Borrow data for this field from the deserializer by using zero-copy
-  deserialization. See [this example](borrow.md).
+  deserialization. See [this example](lifetimes.md#borrowing-data-in-a-derived-impl).
```

Comment:
The broken link takes you to https://serde.rs/borrow.md, which doesn't exist.

Going to https://serde.rs/borrow.html redirects to https://serde.rs/lifetimes.html#borrowing-data-in-a-derived-impl.

I tried out the `lifetimes.md#borrowing-data-in-a-derived-impl` in VS Code and it took me to the correct spot, so I'm hoping it works in GitBook as well 🤞